Source/WebCore/dom/Element.cpp

```diff
@@ -3260,11 +3290,12 @@ ExceptionOr<ShadowRoot&> Element::attachShadow(const ShadowRootInit& init, Custo
         WTFMove(registry), scopedRegistry);
     if (registryKind == CustomElementRegistryKind::Null)
         shadow->setUsesNullCustomElementRegistry(); // Set this flag for Element::insertedIntoAncestor.
+    shadow->setReferenceTarget(AtomString(init.referenceTarget));
```

Comment:
Why is this not part of `ShadowRoot::create` like the other init options?

Reply:
I was hesitant to add a param to `ShadowRoot::create` and the constructor for a currently-experimental feature, but I guess that ship has sailed when the param has already been threaded all the way through to here...